Src/Helpers/domInteractions.ts

```diff
@@ -1,48 +1,51 @@
-import { randomIntFromInterval } from "./mathHelper.js"
+// eslint-disable-next-line import/extensions
+import { randomIntFromInterval } from './mathHelper.js'
 
 let parentDiv:HTMLDivElement
 
-export function initialize(newDiv:HTMLDivElement){
-    parentDiv = newDiv
+export function initialize (newDiv:HTMLDivElement) {
+  parentDiv = newDiv
 }
 
-export function updateCirclePositionInDom(circleDom:HTMLDivElement,x:number,y:number){
-    circleDom.style.left=x+"px"
-    circleDom.style.top=y+"px"
+export function updateCirclePositionInDom (circleDom:HTMLDivElement, x:number, y:number) {
+  // eslint-disable-next-line no-param-reassign
+  circleDom.style.left = `${x}px`
+  // eslint-disable-next-line no-param-reassign
+  circleDom.style.top = `${y}px`
 }
 
-export function addCircleToDom(circleDom:HTMLDivElement){
-    parentDiv.append(circleDom)
+export function addCircleToDom (circleDom:HTMLDivElement) {
+  parentDiv.append(circleDom)
 }
 
-export function removeCircleFromDom(circleDom:HTMLDivElement){
-    parentDiv.removeChild(circleDom)
+export function removeCircleFromDom (circleDom:HTMLDivElement) {
+  parentDiv.removeChild(circleDom)
 }
 
-export function updateCircleRadiusInDom(circleDom:HTMLDivElement,newRadius:number){
-    circleDom.style.width = newRadius+"px"
-    circleDom.style.height = newRadius+"px"
+export function updateCircleRadiusInDom (circleDom:HTMLDivElement, newRadius:number) {
+  // eslint-disable-next-line no-param-reassign
```

Comment:
Disabling rules is not ideal. Seems like this rule can be configured to allow this use case. See details here: https://eslint.org/docs/rules/no-param-reassign#props